README.md

```diff
@@ -47,15 +51,18 @@ When changing EigenLayer contracts, the anvil state needs to be updated with:
 make anvil-deploy-eigen-contracts
 ```
 
-When changing AlignedLayer contracts, the anvil state needs to be updated with:
```

Comment:
This comment shouldn't have been deleted, right ?

Reply:
It was moved down, not deleted